doc/source/user_guide/timeseries.rst

```diff
@@ -1981,6 +1981,9 @@ As with ``DatetimeIndex``, the endpoints will be included in the result. The exa
 
    dfp['2013-01-01 10H':'2013-01-01 11H']
 
+As of version 1.1.0, this works with non-monotonic ``PeriodIndex`` indexes.
```

Comment:
ok, should be versionadded directive. (ad put at the beginning of the slicing with period strings section); thought we didn't have *any* docs on this (but seems we do)